### PR TITLE
Add Google ACME cert renewal specific directions.

### PR DESCRIPTION
**Pull request by banaag**
Changes proposed in this pull request:
- Updating instructions to include Google CA specific details.

### CODE REVIEW

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md [hunk 1]
```diff
@@ -317,6 +317,12 @@ used with signed exchanges.
 *  You have a `CanSignHttpExchanges` certificate. This
      [page](https://github.com/google/webpackager/wiki/Certificate-Authorities)
      lists the CAs that offer this type of certificate.
+   
+*  If you don't have a certificate but have access to automated certificate renewal using
+   ACME, then you can configure your webpkgserver to automatically retrieve
```
Comment (twifkak) [inline src/site/content/en/fast/signed-exchanges-webpackager/index.md:322]: How does one know if they have access to automated certificate renewal using ACME? I'm assuming that's almost everybody, so maybe simplify the sentence to:

> If you don't have a certificate, then you can configure your webpkgserver to automatically retrieve certificates from your CA using ACME.

**Review by twifkak — COMMENTED** [review]
Thanks for the update! LGTM with nit.

File: src/site/content/en/fast/signed-exchanges-webpackager/index.md (hunk 1 above)
Comment (banaag) [reply src/site/content/en/fast/signed-exchanges-webpackager/index.md]: Done.

Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges-webpackager/index.md]: LGTM!

**Review by malchata — APPROVED** [review]
I'm not an SXG subject matter expert, so I'm implicitly trusting that this is accurate. From a style point of view, LGTM!

**Pull request merged**